p2p/ipld/sample.go

```diff
@@ -0,0 +1,102 @@
+package ipld
+
+import (
+	"github.com/ipfs/go-cid"
+	"github.com/lazyledger/nmt/namespace"
+
+	"github.com/lazyledger/lazyledger-core/libs/rand"
```

Comment:
As far as I remember this uses math/rand internally (seeded with crypto/rand or something like this). Here we should rather use crypto/rand to generate the row/col indices.

Reply:
Ok